test/test.js

```diff
@@ -72,23 +72,22 @@ test('strings', t=>{
         jams`{{bad key} val}`
     })
 
-    o = jams(`""`)
-    t.equal("", o) //err, quotes are being escaped for some reaosn
+    t.throws(_ => jams(""))
 
     o = jams(`{key "val"}`)
     t.equal(o.key, "val")
 
-    o = jams(`{\"key \"val}`)
-    t.equal(o[`"key`], `"val`)
+    o = jams(`{\"key \" val}`)
+    t.equal(o[`key `], `val`)
 
-    o = jams(`{\\key val}`)
+    t.throws(_ => jams(`{"key\\" val}`))
```

Comment:
I had to remind myself backslashes aren't accepted by our grammar, if I'm not wrong.

Reply:
There is extra confusion here because `` is not quite raw string and still has escapes, we should just use files for all test cases that deal with escapes